recipes/tokyocabinet/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+./configure --prefix=$PREFIX \
+	    --with-zlib=$PREFIX \
+	    --with-bzip=$PREFIX
+
+make
+make check > /dev/null
```

Comment:
It's not breaking the CIs, but their web interface. I think travis just truncates the output after so many lines. Any error that occurred after that is completely hidden. With the redirections we still see the stderr of the tests and also the message that tests failed.